src/pages/docs/webhook-directory.md

```diff
@@ -56,8 +56,9 @@ New webhooks are created and improved every day. Please [contribute](/docs/how-t
   {% webhook-entry provider="Tiktok" hash="sha256" encode="hex" rotation="❌" versioning="❌" timestamp="✅" link="https://developers.tiktok.com/doc/webhooks-verification" /%}
   {% webhook-entry provider="TrendMicro Conformity" hash="sha256" encode="hex" rotation="❌" versioning="❌" timestamp="❌" link="https://cloudone.trendmicro.com/docs/conformity/webhook-communication/" /%}
   {% webhook-entry provider="Twilio" hash="sha1" encode="base64" rotation="❌" versioning="❌" timestamp="❌" link="https://www.twilio.com/docs/usage/security#validating-requests" /%}
+  {% webhook-entry provider="Twitter Account Activity API" hash="SHA265" encode="base64" rotation="❌" versioning="❌" timestamp="✅" link="https://developer.twitter.com/en/docs/twitter-api/premium/account-activity-api/overview" /%}
```

Comment:
Hi @JessicaGarson. Thanks for contributing <3 
Can we change the hash to `sha256`? 
(the rest looks good)

Reply:
Yes!

Reply:
Just made that change

Reply:
Hi @JessicaGarson . It seems like you replaced the wrong field (encode) with sha256:

```
{% webhook-entry provider="Twitter Account Activity API" hash="SHA265" encode="sha256" rotation="❌" versioning="❌" timestamp="✅" link="https://developer.twitter.com/en/docs/twitter-api/premium/account-activity-api/overview" /%}
```

I should look like this:

```
{% webhook-entry provider="Twitter Account Activity API" hash="sha256" encode="base64" rotation="❌" versioning="❌" timestamp="✅" link="https://developer.twitter.com/en/docs/twitter-api/premium/account-activity-api/overview" /%}
```